session1/3-challenge/3_simple_logic.rb

```diff
@@ -8,4 +8,12 @@
 # ten_twenty(6) # => 10
 
 def ten_twenty(n)
+	if n.even?
+		10
+	elsif n.odd?
```

Comment:
George,

There are no numbers that are not even or odd - if you are thinking of zero, it is even :), the second limb of the conditional is not required:

def ten_twenty(n)
  if n.even?
   10
 else
  20
 end

Reply:
Spot on again